docs/_utils/redirects.yaml

```diff
@@ -0,0 +1,128 @@
+/stable/design-notes/migrating-from-users-to-roles.html: https://github.com/scylladb/scylla/blob/master/docs/dev/migrating-from-users-to-roles.md
+/stable/design-notes/paged-queries.html: https://github.com/scylladb/scylla/blob/master/docs/dev/paged-queries.md
+/stable/design-notes/protocol-extensions.html: https://github.com/scylladb/scylla/blob/master/docs/dev/protocol-extensions.md
+/stable/design-notes/protocols.html: https://github.com/scylladb/scylla/blob/master/docs/dev/protocols.md
+/stable/design-notes/redis.html: https://github.com/scylladb/scylla/blob/master/docs/dev/redis.md
+/stable/design-notes/repair_based_node_ops.html: https://github.com/scylladb/scylla/blob/master/docs/dev/repair_based_node_ops.md
+/stable/design-notes/reverse-reads.html: https://github.com/scylladb/scylla/blob/master/docs/dev/reverse-reads.md
+/stable/design-notes/row_cache.html: https://github.com/scylladb/scylla/blob/master/docs/dev/row_cache.md
+/stable/design-notes/row_level_repair.html: https://github.com/scylladb/scylla/blob/master/docs/dev/row_level_repair.md
+/stable/design-notes/secondary_index.html: https://github.com/scylladb/scylla/blob/master/docs/dev/secondary_index.md
+/stable/design-notes/sstable-scylla-format.html: https://github.com/scylladb/scylla/blob/master/docs/dev/sstable-scylla-format.md
+/stable/design-notes/sstables-directory-structure.html: https://github.com/scylladb/scylla/blob/master/docs/dev/sstables-directory-structure.md
+/stable/design-notes/system_keyspace.html: https://github.com/scylladb/scylla/blob/master/docs/dev/system_keyspace.md
+/stable/design-notes/system_schema_keyspace.html: https://github.com/scylladb/scylla/blob/master/docs/dev/system_schema_keyspace.md
+/stable/design-notes/wasm.html: https://github.com/scylladb/scylla/blob/master/docs/dev/wasm.md
+/stable/guides/index.html: https://github.com/scylladb/scylla/blob/master/docs/dev/index.md
+/stable/guides/HACKING.html: https://github.com/scylladb/scylla/blob/master/HACKING.md
+/stable/guides/api_v2.html: https://github.com/scylladb/scylla/blob/master/docs/dev/api_v2.md
+/stable/guides/building.html: https://github.com/scylladb/scylla/blob/master/docs/dev/building.md
+/stable/guides/debugging.html: https://github.com/scylladb/scylla/blob/master/docs/dev/debugging.md
+/stable/guides/docker-hub.html: https://github.com/scylladb/scylla/blob/master/docs/dev/docker-hub.md
+/stable/guides/logging.html: https://github.com/scylladb/scylla/blob/master/docs/dev/logging.md
+/stable/guides/testing.html: https://github.com/scylladb/scylla/blob/master/docs/dev/testing.md
+/stable/guides/tracing.html: https://github.com/scylladb/scylla/blob/master/docs/dev/tracing.md
+/stable/service_levels.html: https://github.com/scylladb/scylla/blob/master/docs/dev/service_levels.md
+
+# 4.6 branch
+/branch-4.6/contribute/index.html: https://github.com/scylladb/scylla/blob/master/docs/dev/index.md
```

Comment:
This redirects from branch 4.6 to master. Is this what we want to do?

Reply:
Good catch! I'm redirecting now to the specific branch.